test/integration/test_open_analyse_save_load_project.c

```diff
@@ -37,7 +37,7 @@ bool test_open_analyse_save() {
 	size_t functions_count_expect = rz_list_length(functions);
 
 	// 4. Save into the project
-	if (!rz_file_exists(".tmp/")) {
+	if (!rz_file_is_directory(".tmp/")) {
```

Comment:
same, question. isn't illegal this folder from windows?
```suggestion
	if (!rz_file_is_directory(".tmp" RZ_SYS_DIR)) {
```